staging/src/k8s.io/client-go/testing/fixture.go

```diff
@@ -69,6 +69,9 @@ func ObjectReaction(tracker ObjectTracker) ReactionFunc {
 	return func(action Action) (bool, runtime.Object, error) {
 		ns := action.GetNamespace()
 		gvr := action.GetResource()
+		if action.GetSubresource() == "scale" {
+			gvr = schema.GroupVersionResource{Group: "extensions", Version: "", Resource: "scales"}
```

Comment:
This isn't right.  The group cannot be assumed to be extensions.  See RCs and stateful sets as examples.